scripts/validate-changelog.sh

```diff
@@ -11,7 +11,7 @@ package_name="$1"
 shift  # remove package name from arguments
 
 if [[ "${GITHUB_REF:-}" =~ '^release/' ]]; then
-  yarn auto-changelog validate --tag-prefix "${package_name}@" --rc "$@"
+  yarn auto-changelog validate --prettier --tag-prefix "${package_name}@" --rc "$@"
```

Comment:
I think we'll want the same glag added to `update-changelog.sh` as well

Reply:
Yes, good call. Updated in [4c39092](https://github.com/MetaMask/core/pull/3850/commits/4c390925b10973582c768ecc2f087bfe9f72767d).